src/main/java/core/basesyntax/classes/functional/FigureSupplier.java

```diff
@@ -0,0 +1,45 @@
+package core.basesyntax.classes.functional;
+

```

Comment:
Avoid starting the class implementation with an empty line. Remove this redundant empty line as per the checklist guidelines.